src/index.js

```diff
@@ -28,4 +28,60 @@ function renderTasks() {
     taskList.appendChild(listItem);
   });
 }
+
+function addTask(description) {
+  const newTask = {
+    description,
+    completed: false,
+    index: tasks.length,
+  };
+  tasks.push(newTask);
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+  renderTasks();
+}
+
+function deleteTask(index) {
+  tasks.splice(index, 1);
+  // eslint-disable-next-line no-return-assign
+  tasks.forEach((task, i) => (task.index = i));
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+  renderTasks();
+}
+
+function editTask(index, newDescription) {
+  tasks[index].description = newDescription;
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+  renderTasks();
+}
+
+function clearCompletedTasks() {
+  // eslint-disable-next-line no-plusplus
+  for (let i = tasks.length - 1; i >= 0; i--) {
+    if (tasks[i].completed) {
+      tasks.splice(i, 1);
+    }
+  }
+  // eslint-disable-next-line no-return-assign
+  tasks.forEach((task, i) => (task.index = i));
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+  renderTasks();
+}
+
+const form = document.querySelector('form');
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const input = document.getElementById('item');
+  const description = input.value.trim();
+  if (description !== '') {
+    addTask(description);
+    input.value = '';
+  }
+});
```

Comment:
- [ ] Hey, great job with your design. Currently, users can not add any task to your Todo as there's no button to add a task. Even when using the Enter key on the keyboard, nothing gets added. Please, kindly improve your design to improve the user experience of your app. Also, so that your project can be reviewed appropriately. :bowing_woman: 
See the image of your Todo app below:

- [ ] Also, let's try to stick to the design from the video as much as possible by removing the green background color added to the header of the todo list. :+1: 


![Screenshot from 2023-02-25 12-26-33](https://user-images.githubusercontent.com/59397159/221354775-ee0177cb-7b6f-428a-a979-f1643f8e059b.png)

Reply:
### PROJECT REQUIREMENT


- [ ] Hey, according to the [project requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md), please, you should create a new file for the functionality, adding, deleting, editing task. Please, let's try to meet this part of the project requirement.